### PR TITLE
Endpoint that deactivates a users third-party (Google/Github/Twitter) account from atlas admin

### PR DESCRIPTION
**Pull request by Spartakovic**
-Updated User model with new field
-Added endpoint to set and store new field

### CODE REVIEW

**Comment by MartinsIrbe** [comment]
Tests?

**Comment by MartinsIrbe** [comment]
lgtm

File: atlas-api/src/main/java/org/atlasapi/application/users/UsersController.java [hunk 1]
```diff
@@ -146,6 +152,38 @@ public void updateUser(HttpServletRequest request,
             new ErrorResultWriter().write(summary, writer, request, response);
         }
     }
+
+    @RequestMapping(value = "/4/users/deactivate/{uid}.*", method = RequestMethod.POST)
```
Comment (MartinsIrbe) [inline atlas-api/src/main/java/org/atlasapi/application/users/UsersController.java]: Would be better -  4/users/{uid}.*/deactivate

File: atlas-api/src/main/java/org/atlasapi/application/users/UsersController.java [hunk 2]
```diff
@@ -146,6 +152,38 @@ public void updateUser(HttpServletRequest request,
             new ErrorResultWriter().write(summary, writer, request, response);
         }
     }
+
+    @RequestMapping(value = "/4/users/{uid}/deactivate.*", method = RequestMethod.POST)
```
Comment (emilssolmanis) [inline atlas-api/src/main/java/org/atlasapi/application/users/UsersController.java]: We've agreed to not use verbs in paths, so this needs to be fixed.

Comment (Spartakovic) [inline atlas-api/src/main/java/org/atlasapi/application/users/UsersController.java]: I am just updating one field, but don't delete anything, does it make any difference? I am not sure of the semantics of "DELETE", I thought it is applied only when we actually delete something. I can update it

Comment (emilssolmanis) [inline atlas-api/src/main/java/org/atlasapi/application/users/UsersController.java]: @Spartakovic it's the outside semantics that matter, not the underlying data model. @Dalamar42 @tfm what do you think?

**Pull request merged**